dropwizard-unixsocket/src/main/java/io/dropwizard/unixsocket/UnixSocketFactory.java

```diff
@@ -0,0 +1,70 @@
+package io.dropwizard.unixsocket;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.dropwizard.jetty.HttpConnectorFactory;
+import org.eclipse.jetty.io.ByteBufferPool;
+import org.eclipse.jetty.server.ConnectionFactory;
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.unixsocket.UnixSocketConnector;
+import org.eclipse.jetty.util.thread.Scheduler;
+import org.eclipse.jetty.util.thread.ThreadPool;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+/**
+ * Builds Unix Domain Socket connectors.
+ * <p/>
+ * <b>Configuration Parameters:</b>
+ * <table>
+ *     <tr>
+ *         <td>Name</td>
+ *         <td>Default</td>
+ *         <td>Description</td>
+ *     </tr>
+ *     <tr>
+ *         <td>{@code socketFilename}</td>
+ *         <td></tmp/dropwizard.sock</td>
+ *         <td>
+ *             The filename of the opened Unix socket on the filesystem.
+ *         </td>
+ *     </tr>
+ * </table>
+ * <p/>
+ */
+@JsonTypeName("unixsocket")
+public class UnixSocketFactory extends HttpConnectorFactory {
+
+    private String socketFilename = "/tmp/dropwizard.sock";
```

Comment:
Should we prefer `/var/run/dropwizard.sock` ([Stackoverflow](http://stackoverflow.com/q/7580346))?